sdk/core/core-tracing/lib/interfaces/plugin.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Tracer } from "./tracer";
+import { SupportedPlugins } from "../utils/supportedPlugins";
+
+export interface Plugin extends Tracer {
```

Comment:
What was the motivation to introduce the `Plugin`?

Reply:
Exactly. And I think we can do away with it soon (next PR!) so it will not trouble us for long. 

My ultimate goal is that a developer already using OpenTelemetry can just pass us the Tracer they want us to use and it works. For OpenCensus users, they can wrap it in our facade and do the same.